war/src/main/js/keyboard-shortcuts.js

```diff
@@ -12,6 +12,16 @@ window.addEventListener("load", () => {
     // Returning false stops the event and prevents default browser events
     return false;
   });
+
+  const pageSearchBar = document.querySelectorAll(".jenkins-search__input");
+  if (pageSearchBar.length === 1) {
+    hotkeys(translateModifierKeysForUsersPlatform("/"), () => {
```

Comment:
If I press my key combination to generate a slash, I have no selection or focus. Am I missing something?

I used the following snipped to check whether my input is considered:

```patch
--- a/war/src/main/js/keyboard-shortcuts.js
+++ b/war/src/main/js/keyboard-shortcuts.js
@@ -15,7 +15,9 @@ window.addEventListener("load", () => {
 
   const pageSearchBar = document.querySelectorAll(".jenkins-search__input");
   if (pageSearchBar.length === 1) {
-    hotkeys(translateModifierKeysForUsersPlatform("/"), () => {
+    let input = "t"
+    hotkeys(translateModifierKeysForUsersPlatform(input), () => {
+      console.log(input)
       pageSearchBar[0].focus();
```

My input with "/" wasn't considered at all, I had to change it to something else. I'm using a letter as a working example.

Reply:
In terms of https://github.com/jenkinsci/jenkins/pull/7074/commits/ff4d6851bd7cab039b70cabc987acecd1e45cd35, what kind of keyboard do you use, where CMD is used for a slash @timja 👀 
Isn't that typically done via Shift (and equally shift+option for a backslash), at least that's the case on my macbook.

Reply:
Updated this to include a singular slash without CMD @timja - is that all good with you?

Reply:
Hmm the singular / didn’t work for me in chrome or safari on Mac

Reply:
hmm it's working on Safari and firefox for me but not Chrome

Reply:
Ok it's working for me now, I assume it was cached locally. I've reverted the commits